src/main/kotlin/io/github/gradlenexus/publishplugin/NexusPublishPlugin.kt

```diff
@@ -317,3 +317,8 @@ class NexusPublishPlugin : Plugin<Project> {
         }
     }
 }
+
+inline fun <reified T : Any> Project.theExtension(): T =
```

Comment:
This issue in gradle https://github.com/gradle/gradle/issues/25206 causes a deprecation warning when using the `the` extension method to fetch extensions. This would cause tests to fail when running on gradle 8.2-milestoneX

Removing usage of the `the` method fixes this